flintrock/flintrock.py

```diff
@@ -334,6 +339,22 @@ def launch(
             )
         services += [spark]
 
+    # Format ec2 tags
+    if ec2_tags:
+        for ec2_tag in ec2_tags:
+            if ec2_tag.count(',') != 1:
+                raise UsageError(
+                    "Error: EC2 tags need to be specified as Key,Value pairs "
+                    "separated by a single comma")
+            k, v = ec2_tag.split(',')
+            if not len(k) or not len(v):
+                raise UsageError(
+                    "Error: An EC2 tag seems to have a missing key or value")
```

Comment:
Is a missing value a problem? I would assume that you can tag something with a key but no value.